arbiter-core/src/data_collection.rs

```diff
@@ -184,12 +162,70 @@ impl EventLogger {
     /// This function will return an error if there is a problem creating the
     /// directories or files, or writing to the files.
     pub fn run(self) -> Result<(), RevmMiddlewareError> {
-        tokio::spawn(async move {
-            let mut set = self.events;
-            while let Some(res) = set.join_next().await {
-                info!("task completed: {:?}", res);
+        let receiver = self.receiver.unwrap();
+        let dir = self.directory.unwrap_or("./out".into());
```

Comment:
Does this overwrite if an existing directory is there? Might conflict with default foundry output which is `out`, if it does an it overwrites the foundry out that could make issues

Reply:
Good catch. @kinrezC let's change this do something like `./data_output`